File: TUF/Numbers/PrintFibboNthTerm.java
import java.util.Scanner;

public class PrintFibboNthTerm {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the nth term value: ");
        int n = input.nextInt();
        input.close();
        System.out.println(fibonacci(n));
    }

    private static int fibonacci(int n) {
        if (n <= 1) {
            return n;
        } else {
            int last = fibonacci(n - 1);
            int slast = fibonacci(n - 2);
            return last + slast;
        }
    }
}

File: TUF/Numbers/PrintFibboNthTerm.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
function fibonacci(n) {
    if (n <= 1) {
        return n;
    }
    else {
        var last = fibonacci(n - 1);
        var slast = fibonacci(n - 2);
        return last + slast;
    }
}
var n = 6;
console.log(fibonacci(n));
